pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdFunctions.java

```diff
@@ -240,6 +240,8 @@ void processArguments() throws Exception {
         protected FunctionConfig.ProcessingGuarantees processingGuarantees;
         @Parameter(names = "--userConfig", description = "User-defined config key/values")
         protected String userConfigString;
+        @Parameter(names = "--retainOrdering", description = "Function consumes and processes messages in order")
```

Comment:
Not sure if the flag makes sense at this level

Reply:
Do we think that actually specifying subscription type might be the way to go here. We used to have that flag but removed it.

Reply:
> We used to have that flag but removed it.

hmm.. not sure about it but it requires to manage ordering.